content/docs/1.6.0/advanced-resources/os-distro-specific/talos-linux-support.md

```diff
@@ -60,6 +60,41 @@ For detailed instructions, see the Talos documentation on [Editing Machine Confi
 
 - Exclusive to v1 data volume: currently, within a Talos Linux cluster, Longhorn only supports v1 data volume. The v2 data volume isn't currently supported in this environment.
 
+## Talos Linux Upgrades
+
+When [upgrading a Talos Linux node](https://www.talos.dev/v1.7/talos-guides/upgrading-talos/#talosctl-upgrade), always include the `--preserve` option in the command. This option explicitly tells Talos to keep ephemeral data intact.
+
+Example:
+
+```
+talosctl upgrade --nodes 10.20.30.40 --image ghcr.io/siderolabs/installer:v1.7.6 --preserve
+```
+
+> **Caution:**
+> If you do not include the `--preserve` option, Talos wipes `/var/lib/longhorn`, destroying all replicas stored on that node.
```

Comment:
The other option is to suggest that users set up a dedicated partition of a disk as a Longhorn disk. This will be soon used in v2 adoption.

https://www.talos.dev/v1.7/reference/configuration/v1alpha1/config/#Config.machine.disks.

@c3y1huang can you improve the doc later?

Reply:
Remove the commits from this PR because they are not in the scope of the PR.